scripts/generateAssetData/generateAssetData.ts

```diff
@@ -170,10 +170,13 @@ const main = async () => {
     await generateRelatedAssetIndex()
 
     console.info(
-      "Done. Don't forget to add a migration to clear assets state so the new assets are loaded.",
+      "Assets and related assets data generated. Don't forget to add a migration to clear assets state so the new assets are loaded.",
```

Comment:
Same console.info in `generateAssetData` and `generateRelatedAssetIndex` was off and a pain for debugging.